cldf/cldf-metadata.json

```diff
@@ -0,0 +1,362 @@
+{
+    "@context": "http://www.w3.org/ns/csvw",
+    "aboutUrl": null,
+    "dc:bibliographicCitation": "Daniel et al. (2021) Lingua Franca as Lexical Donors: Evidence from Daghestan. Language.",
+    "dc:conformsTo": "http://cldf.clld.org/v1.0/terms.rdf#Wordlist",
+    "dc:isVersionOf": null,
+    "dc:license": "https://creativecommons.org/licenses/by/4.0/",
+    "dc:related": null,
+    "dc:source": "sources.bib",
+    "dc:title": "CLDF dataset accompanying Daniel et al.'s \"Lingua Franca as Lexical Donors: Evidence from Daghestan\" from 2021",
+    "dcat:accessURL": "git@github.com:ilchec/daniellinguafrancas",
+    "prov:wasDerivedFrom": [
+        {
+            "rdf:type": "prov:Entity",
+            "dc:title": "Repository",
+            "rdf:about": "git@github.com:ilchec/daniellinguafrancas",
+            "dc:created": "428a3d0"
+        },
+        {
+            "rdf:type": "prov:Entity",
+            "dc:title": "Glottolog",
+            "rdf:about": "https://github.com/glottolog/glottolog",
+            "dc:created": "v4.1-20-gea2ca69206"
```

Comment:
This shows that you apparently use a version of glottolog that is after 4.1, but not 4.3, so it also shows that the glottolog version was not specified when creating the cldf dataset. I guess, you should go to your glottolog repository and `git pull` to have the most recent version. Could you do so (also with conceepticon) and check again?